README.md

```diff
@@ -1,6 +1,6 @@
 # Skbn
 
-Skbn is a tool for copying files and directories between Kubernetes and AWS S3. It is named after the 1981 video game [Sokoban](https://en.wikipedia.org/wiki/Sokoban).
+Skbn is a tool for copying files and directories between Kubernetes, AWS S3 or Azure Blob Storage. It is named after the 1981 video game [Sokoban](https://en.wikipedia.org/wiki/Sokoban).
```

Comment:
and, not or :)
This is because you can actually copy files from S3 to ABS (and vice versa).